src/overlays/actors/ovl_En_Kakasi/z_en_kakasi.h

```diff
@@ -6,9 +6,28 @@
 
 struct EnKakasi;
 
+typedef void (*EnKakasiFunc)(struct EnKakasi*, GlobalContext*);
+
 typedef struct EnKakasi {
     /* 0x0000 */ Actor actor;
-    /* 0x014C */ char unk_14C[0xC0];
+    /* 0x014C */ EnKakasiFunc actionFunc;
+    /* 0x0150 */ SkelAnime skelanime;
+    /* 0x0194 */ u8 unk_194;
+    /* 0x0195 */ char unk_195;
+    /* 0x0196 */ s16 unk_196;
+    /* 0x0198 */ s16 unk_198;
+    /* 0x019A */ s16 unk_19A;
+    /* 0x019C */ s16 unk_19C[4];
+    /* 0x01A4 */ s16 unk_1A4;
+    /* 0x01A6 */ s16 unk_1A6;
+    /* 0x01A8 */ s16 unk_1A8;
+    /* 0x01AA */ char unk_1AA[0x2];
```

Comment:
This seems to be required to match

Reply:
It probably is an unused s16, it can't be 2 bytes of padding in the middle of s16s
leaving it as char[] is ok